--- a/src/compat/compat.h
+++ b/src/compat/compat.h
@@ -683,6 +683,9 @@ static inline void *skb_put_zero(struct sk_buff *skb, unsigned int len)
 	return ret;
 }
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 13, 0)
+// skb_put_data is already available in kernel 4.13+
+#else
 static inline void *skb_put_data(struct sk_buff *skb, const void *data, unsigned int len)
 {
 	void *ret = skb_put(skb, len);
@@ -690,6 +693,7 @@ static inline void *skb_put_data(struct sk_buff *skb, const void *data, unsigned
 	return ret;
 }
+#endif
 
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 10, 0)
 // timer_setup is already available in kernel 4.10+
