--- a/src/compat/compat.h
+++ b/src/compat/compat.h
@@ -680,8 +680,11 @@ static inline void *skb_put_zero(struct sk_buff *skb, unsigned int len)
 	return ret;
 }
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 13, 0)
+/* skb_put_data is only available in kernel 4.13+ */
 static inline void *skb_put_data(struct sk_buff *skb, const void *data, unsigned int len)
 {
 	void *ret = skb_put(skb, len);
 	memcpy(ret, data, len);
 	return ret;
 }
+#endif
 
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 10, 0)
